ITEM.tsv

```diff
@@ -256,7 +256,7 @@ ITEM_20150317_000255	Meteor Rod
 ITEM_20150317_000256	Ring Rod
 ITEM_20150317_000257	Servant Rod
 ITEM_20150317_000258	Panto Rod
-ITEM_20150317_000259	Wizards have been discovering new facts from monsters emerging from the Great Plant Cataclysm and applying for various use.
+ITEM_20150317_000259	Wizards have been discovering new facts from monsters emerging from the Great Plant Cataclysm applying it for various use.
```

Comment:
"It" would be implying a singular term being applied, rather "and applying them to varying situations." Sounds cleaner and implies the facts (since its plural) are being put to use. Even if the translation is a little off, using the word use at the end of the sentence doesn't sound correct and using uses is incorrect.